### PR TITLE
librsvg: update to 2.58.1.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture: **x86_64**

### CODE REVIEW

File: srcpkgs/librsvg/template [hunk 1]
```diff
@@ -14,9 +14,9 @@ maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/LibRsvg"
 # update changelog when release series changes
-changelog="https://gitlab.gnome.org/GNOME/librsvg/-/raw/librsvg-2.57/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/librsvg/-/raw/librsvg-2.58/NEWS"
```
Comment (tranzystorekk) [inline srcpkgs/librsvg/template:17]: could automate this with `${version%.*}`

Comment (icp1994) [reply srcpkgs/librsvg/template:17]: I think the informal policy is to have no variable in `changelog`

Comment (tranzystorekk) [reply srcpkgs/librsvg/template:17]: I'm fine either way but it feels like duplicating the version here is more error-prone

Comment (Piraty) [reply srcpkgs/librsvg/template:17]: @icp1994 please use the variable

Comment (icp1994) [reply srcpkgs/librsvg/template:17]: Are you sure though? There has been strong pushback against using variables in places like homepage, changelog etc by other maintainers in the past.

Comment (Piraty) [reply srcpkgs/librsvg/template:17]: i don't see validity of "strong pushback" claim. The only thing in this regard i can remember was https://github.com/void-linux/void-packages/issues/27735 and yes, it's still unsettled

* `homepage` is usually free of variables for parsing reasons (i think)
* `distfiles` don't make sense without variables (especially `$version`)
* `changelog` is preferred to point to the active branch if there is one. if there is no such thing, the next best thing is to point to the versioned hyperlink.

if not following an idiom (regardless of its justification) leads to less error-prone maintenance overhead, do it.

Comment (icp1994) [reply srcpkgs/librsvg/template]: Fine by me then.

**Pull request merged**